src/MICore/LaunchOptions.cs

```diff
@@ -551,6 +553,15 @@ private static string ResolveFromPath(string command)
                 {
```

Comment:
I think you may have missed my previous question: do we actually want to check for `gdb` (no extension) on Windows? I would have thought that that wouldn't work.

Reply:
I modified my `MinGW\bin\gdb.exe` to be `MinGW\bin\gdb` and it still works.